tests/suites/cloud/suite.js

```diff
@@ -275,22 +275,56 @@ module.exports = {
     await this.worker.on();
 
     // create tunnels
-    this.log('Creating SSH tunnels to DUT');
-    await this.worker.createSSHTunnels(
-      this.link
-    );
+		test.comment(`Creating SSH tunnels to DUT - trying to find IP of ${this.link}`);
+		await test.resolves(
+			this.worker.createSSHTunnels(
+				this.link,
+			),
+			`Should detect ${this.link} on local network and establish tunnel`
+		)
 
     this.log('Waiting for device to be reachable');
-    await this.utils.waitUntil(async () => {
-      this.log("Trying to ssh into device");
-      let hostname = await this.context
-        .get()
-        .worker.executeCommandInHostOS(
-          "cat /etc/hostname",
-          this.link
-        )
-      return (hostname === `${this.balena.uuid.slice(0, 7)}`)
-    }, true, 60, 5 * 1000);
+    await test.resolves(
+			this.utils.waitUntil(async () => {
+				test.comment(`Trying to SSH into device ${this.link}`);
+				let hostname = await this.context
+				.get()
+				.worker.executeCommandInHostOS(
+				"cat /etc/hostname",
+				this.link
+				)
+				return (hostname === this.link.split('.')[0])
+			}, true),
+			'Device should be reachable over local SSH connection'
+		)
```

Comment:
This block of tests could probably be shorter and easier to reason about.

This is subjective, but something like:
```
this.utils.waitUntil(
    this.worker.executeCommandInHostOS(
        '...', this.link
    ).then(res => res == this.link.split('.')[0])
)
```

I haven't tested the above, it may still require wrapping `executeCommandInHostOS` in a function.

Reply:
the only problem I have with this suggestion is that we start mixing async/await with .then() styling - again, subjective here, but it starts to get quite confusing to me when going through this. Then again, we already have these mixed around throughout the suite...